### DISCUSSION TITLE
Issue with "Smart Filter" Feature

### DISCUSSION THREAD

**Discussion by Bog555** [Authenticator-Extension/Authenticator#1099] [Q&A]
I am writing to address a consistent issue I have encountered with the Google Authenticator extension across multiple Chrome browsers.

On one of my computers, the Authenticator extension displays a "Smart Filter" search field. Interestingly, on another computer with the exact same version of both Chrome and the Authenticator extension, this search field is not present.

This inconsistency has made the user experience confusing, and I'm unsure of how to manage this feature. While I attempted to remove the "Smart Filter" from one browser, I found myself unable to either disable or re-enable it afterward.

Could you provide some clarity on why there might be a discrepancy between the two installations, even though they're on the same versions? Additionally, guidance on how to consistently enable or disable the "Smart Filter" feature would be greatly appreciated.

**Comment by mymindstorm** [comment]
Do you have a screenshot of the "smart filter" you are referring to?
